sklearn/ensemble/_stacking.py

```diff
@@ -326,6 +346,11 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validation strategies that can be used here.
 
+        If “prefit” is passed, it is assumed that all base_estimators have
```

Comment:
The parameter in this class is called `estimators`:

```suggestion
        If “prefit” is passed, it is assumed that all `estimators` have
```

Also lets move this paragraph above the `"Refer User Guide" paragraph, so that User guide link is at the end.

To be complete, we can include "prefit" as an option in the list:

https://github.com/scikit-learn/scikit-learn/blob/1d1aadd0711b87d2a11c80aad15df6f8cf156712/sklearn/ensemble/_stacking.py#L317